framework/src/transfers/MultiAppConservativeTransfer.C

```diff
@@ -114,43 +114,45 @@ MultiAppConservativeTransfer::initialSetup()
   {
     if (_from_postprocessors_to_be_preserved.size() == 1 && _current_direction == TO_MULTIAPP)
     {
-      FEProblemBase & from_problem = _multi_app->problemBase();
+      FEProblemBase & from_problem = _to_multi_app->problemBase();
```

Comment:
Should be `_from_multi_app` ?

Reply:
no. We get the problemBase here, so we get the main app's problem essentially. 
Since this is a TO case, _from_multiapp does not exist. But _to_multiapp can allow us to retrieve the main problem, which is the from_problem